### PR TITLE
#165413090 Setup Travis CI for management service

### PR DESCRIPTION
**Pull request by AJAkimana**
#### Description
Stable CI setup will make it easy to integrate our code while ensuring that all defined tests are passing per integration.
#### Type of change
N/A
#### How Has This Been Tested?
N/A
#### Checklist:
N/A
#### PIVOTAL TRACKER
[#165413090](https://www.pivotaltracker.com/n/projects/2326984/stories/165413090)

### CODE REVIEW

**Review by ericnyirimana — COMMENTED** [review]
You have set up well TravisCI. The remain thing is to merge with @jabichris test so that the test can be built successfully.

**Comment by Kabalisa** [comment]
@AJAkimana Travis is configured well but the Acceptance Criteria title in your PR is not included in the format of how to write a PR in the engineering playbook.

**Review by Kabalisa — CHANGES_REQUESTED** [review]
@AJAkimana Travis is configured well but the Acceptance Criteria title in your PR is not included in the format of how to write a PR in the engineering playbook.

**Comment by AJAkimana** [comment]
> @AJAkimana Travis is configured well but the Acceptance Criteria title in your PR is not included in the format of how to write a PR in the engineering playbook.

Ok I have edited it

**Review by jabichris — COMMENTED** [review]
You need tests.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - "stable"
+
```
Comment (macphilips) [inline .travis.yml:6]: remove extra new line

**Review by macphilips — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (AJAkimana) [reply .travis.yml:6]: > remove extra new line

I removed it

Comment (macphilips) [reply .travis.yml:6]: I can still see it here

Comment (AJAkimana) [reply .travis.yml:6]: I did correct it. Check it again

**Pull request merged**